doc/classes/Performance.xml

```diff
@@ -123,105 +123,108 @@
 	</methods>
 	<constants>
 		<constant name="TIME_FPS" value="0" enum="Monitor">
-			The number of frames rendered in the last second. This metric is only updated once per second, even if queried more often. [i]Higher is better.[/i]
+			The number of frames rendered in the last second (identical to [method Engine.get_frames_per_second]). This metric is only updated once per second, even if queried more often. [i]Higher is better.[/i]
 		</constant>
-		<constant name="TIME_PROCESS" value="1" enum="Monitor">
+		<constant name="TIME_FPS_1_PERCENT_LOW" value="1" enum="Monitor">
```

Comment:
I don't think this should be put at the top here but at the end, it disrupts a lot of values here, is there a specific reason for this going at the top?

Reply:
This was a leftover from this PR being made before 4.0's release. I've moved it to the end of the enum.